rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [ ] Getters and Setters for `book` and `person` instance variables should be present. As we are going to `get the value` as well as we need to `set the value`.

Reply:
Fixed